### PR TITLE
Disable pylint error and fix CI tests of new tifffile

### PR DESCRIPTION
**Pull request by Nic-Ma**
### Description
Recently, due to some unclear reason, a pylint error randomly occurred in many CI tests of PRs, for example:
`Catching an exception which doesn't inherit from Exception: ValidationError (catching-non-exception)`
https://github.com/Project-MONAI/MONAI/runs/6075933692?check_suite_focus=true
This PR disabled the error as the workaround.

And also fixed the CI tests error related to the new tifffile released yesterday.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Review by dongyang0122 — APPROVED** [review]

**Comment by Nic-Ma** [comment]
Mark: the root cause of `Tifffile` issue is that: In the new version, if not 3 channel data, it can't save as RGB anymore, if saving (16, 16) shape image as minisblack, it will change to (16, 16, 4) shape. So changed the test case to (16, 16, 2) shape.

Thanks.

File: tests/test_wsireader.py [hunk 1]
```diff
@@ -84,7 +84,7 @@
 
 TEST_CASE_RGB_1 = [np.ones((3, 100, 100), dtype=np.uint8)]  # CHW
 
-TEST_CASE_ERROR_GRAY = [np.ones((16, 16), dtype=np.uint8)]  # no color channel
+TEST_CASE_ERROR_GRAY = [np.ones((16, 16, 2), dtype=np.uint8)]  # wrong color channel
```
[the remark this answers is not in the document]
Comment (drbeh) [reply tests/test_wsireader.py:87]: Thanks

**Pull request merged**